tools/preview/README.md

```diff
@@ -4,10 +4,330 @@ The preview tool for Spectrum CSS is driven by [Storybook](https://storybook.js.
 
 - `assets`: This folder contains static, unprocessed assets for inclusion in the stories.
 - `decorators`: These assets interpret the arguments provided and apply the necessary stylesheet or markup changes to reflect the desired update.
-- `icons.js`: This is a custom file which imports the icon set from `@adobe/spectrum-css-workflow-icons`. It offers an importable icon-type with the a dropdown select list naming each icon and a generateSVG utility to convert that icon name to an SVG element with the appropriate classes.
 
-Storybook files:
-  - `main.js`
-  - `manager.js`
-  - `preview.js`
-  - `preview-head.html` & `main-head.html`
+**Storybook files:**
+
+- `main.js`
+- `manager.js`
+- `preview.js`
+- `preview-head.html` & `main-head.html`
+
+Storybook leverages webpack for bundling and we have updated it with the following settings (these rules are applied to *.stories.js assets as well as the template.js files):
+
+- Packages can be loaded from local or root node_modules directories. i.e., in a story:
+
+  ```js
+  import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+  ```
+
+  or in a template.js file:
+
+  ```js
+  import { Template as Icon } from "@spectrum-css/icon/stories/template.js";
+  ```
+
+- Component CSS can be loaded in a story using the package name rather than the directory path (the local version is used as the path is automatically resolved).
```

Comment:
```suggestion
- CSS for other components can be loaded in a story using the package name (rather than the directory path), i.e. `@spectrum-css/toast` vs. `../toast/index.css`. The local version of the package is used regardless but the webpack settings will resolve the pathing for you.
```

^ What do you think of this @mlogsdon18 as a clarification?